docs/book/appendix-default-aws-settings.md

```diff
@@ -17,6 +17,8 @@ gives this the skill requests's `context.System.device.deviceId` field, by defau
 
 ## Lambda
 
+* Lambda runtime should use `nodejs10.x` or later.
+  * We can check the runtime EOL in [the AWS document](https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html).
```

Comment:
The content is good. Can you reword the sentence to be more specific and consistent with the rest of the docs? For example, we generally try to say 'you' when referring to an action the user can take. You can rewrite it to be `You can check the runtime deprecation schedule in the [Lambda Runtime Support Policy](...)` or something similar. Thank you!